runtimes/runtimes/util/pathUtil.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Simplified version of path.join that can be safely used on web
+ * @param segments
+ * @returns
+ */
+export function joinUnixPaths(...segments: string[]): string {
```

Comment:
"unix" in the name will discourage use for windows. maybe we can just add a `TODO: support "\" slashes` here, to avoid callers thinking they need some windows-specific join
```suggestion
export function joinPaths(...segments: string[]): string {
```

Reply:
This was actually my intention since I don't see a use case for this utility outside of the web environment where we mimic unix paths. Is there a non-web case where we want to use this over `path.join`?

Reply:
Since we need the fix to unblock the web bundling, let's make the changes suggested by @justinmk3 in the followup